e2e/cypress/integration/messaging/markdown_quotation_paragraphs_spec.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18703-Markdown quotation paragraphs', () => {
+        const messageParts = ['this is', 'really', 'three quote lines'];
+
+        cy.visit('/ad-1/channels/town-square');
+
+        // # Post message to use
+        cy.get('#post_textbox').clear().type('>' + messageParts[0]).type('{shift}{enter}{enter}');
+        cy.get('#post_textbox').type('>' + messageParts[1]).type('{shift}{enter}{enter}');
+        cy.get('#post_textbox').type('>' + messageParts[2]).type('{enter}');
```

Comment:
I'm good with this as is but would be nice to access `cy.get('#post_textbox')` once, and chain others as necessary.